services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -866,253 +852,222 @@ private void setUpIndividualRequest(final IIndividualRequest request) {
         final IIndividualRequest individualRequest = new IndividualRequest();
 
-        // Set-up the individual request object
-
         individualRequest.setSdtRequestReference(sdtRequestRef);
         individualRequest.setRequestStatus("Received");
         setUpIndividualRequest(individualRequest);
         individualRequest.setRequestType(JUDGMENT.getType());
         individualRequest.setRequestPayload("Test Xml");
-        expect(xmlReader.getElementValue("Test Xml", "claimNumber")).andReturn("CCD_Reference_number");
-        EasyMock.expectLastCall().times(1);
-        expect(this.ccdReferenceValidator.isValidCCDReference("CCD_Reference_number")).andReturn(true);
-        EasyMock.expectLastCall().times(1);
 
-        expect(this.mockIndividualRequestDao.getRequestBySdtReference(sdtRequestRef)).andReturn(
-            individualRequest);
+        when(this.mockIndividualRequestDao.getRequestBySdtReference(sdtRequestRef)).thenReturn(individualRequest);
 
         final IGlobalParameter individualReqProcessingDelay = new GlobalParameter();
         individualReqProcessingDelay.setValue("10");
         individualReqProcessingDelay.setName("MCOL_INDV_REQ_DELAY");
-        expect(this.mockCacheable.getValue(IGlobalParameter.class, "MCOL_INDV_REQ_DELAY")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, "MCOL_INDV_REQ_DELAY")).thenReturn(
             individualReqProcessingDelay);
 
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
-
         final IGlobalParameter connectionTimeOutParam = new GlobalParameter();
         connectionTimeOutParam.setName("TARGET_APP_TIMEOUT");
         connectionTimeOutParam.setValue("1000");
-        expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_TIMEOUT")).andReturn(
-            connectionTimeOutParam);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_TIMEOUT")).thenReturn(connectionTimeOutParam);
 
         final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
         receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
         receiveTimeOutParam.setValue("12000");
-        expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT")).andReturn(
-            receiveTimeOutParam);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT")).thenReturn(receiveTimeOutParam);
 
-        this.mockCmcConsumerGateway.individualRequest(individualRequest, 1000, 12000);
-        EasyMock.expectLastCall().andAnswer(() -> {
-            throw new CaseOffLineException("Case OffLine");
-        });
-
-        this.mockMessageWriter.queueMessage(EasyMock.isA(ISdtMessage.class), EasyMock.isA(String.class),
-                                            EasyMock.anyBoolean());
-        EasyMock.expectLastCall();
+        this.mockMessageWriter.queueMessage(isA(ISdtMessage.class), isA(String.class), anyBoolean());
```

Comment:
This is line of code is actually calling the queueMessage() method on mockMessageWriter rather than setting up an expected call. This probably needs to be converted to a Mockito verify() and moved to the end of the test where the other verifies are.

(This is in the processCCDReferenceRequestWhenCaseNotOffLine() test.)

Reply:
done